build.sbt

```diff
@@ -4,8 +4,11 @@ version := "1.0-SNAPSHOT"
 
 scalaVersion := "2.11.8"
 
+enablePlugins(GatlingPlugin)
+
 libraryDependencies += "io.gatling" % "gatling-core" % "2.2.0"
 libraryDependencies += "io.gatling" % "gatling-http" % "2.2.0"
-
+libraryDependencies += "io.gatling.highcharts" % "gatling-charts-highcharts" % "2.2.0"
```

Comment:
And I don't see which part of this I am breaking by integrating it in this work : https://raw.githubusercontent.com/gatling/gatling-highcharts/master/LICENSE